src/processout/card.ts

```diff
@@ -151,15 +151,16 @@ module ProcessOut {
             var err = Expiry.validateMonth(this.getMonth());
             if (err)
                 return err;
-            
+
             err = Expiry.validateYear(this.getYear());
             if (err)
                 return err;
-            
+
             var date = new Date();
-            if (this.getMonth() < date.getMonth() && this.getYear() == date.getFullYear())
+            let dateMonth = date.getMonth() + 1;
+            if (this.getMonth() < dateMonth && this.getYear() == date.getFullYear())
```

Comment:
probably there is a smart way to get this correct, but I dont want to force you to look for it :p you can ask copilot though

Reply:
Yeah, we could probably do this "properly", but I took an easy way out 😅